Description: Fix doc-generation on non-linux archs
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2013-04-07

--- minieigen-0.3~dfsg~bzr25.orig/doc/source/conf.py
+++ minieigen-0.3~dfsg~bzr25/doc/source/conf.py
@@ -19,7 +19,7 @@ import sys, os, re, glob
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
 #sys.path.insert(0, os.path.abspath('../../src'))
-sys.path.insert(0,glob.glob(os.path.abspath('../../build/lib.linux-*'))[0])
+sys.path.insert(0,glob.glob(os.path.abspath('../../build/lib.*'))[0])
 print sys.path
 
 
